src/components/PostInfo/PostInfo.scss

```diff
@@ -1 +1,16 @@
-//
+.PostInfo {
+  margin: 10px auto;
+  width: 90%;
+  border: 1px solid #000;
+  border-radius: 8px;
+  background-color: #D3D3D3;
```

Comment:
![image](https://github.com/mate-academy/react_static-list-of-posts/assets/86923089/929393cf-b065-43df-8adc-c4ff8bce21d9)

fix lint